src/borg/testsuite/archiver/__init__.py

```diff
@@ -190,16 +190,20 @@ def open_archive(self, name):
     def open_repository(self):
         return Repository(self.repository_path, exclusive=True)
 
+
+    def assert_line_exists(self, lines, expected_substrs):
+        exists = False
+        for line in lines:
+            if all(substr in line for substr in expected_substrs):
+                exists = True
+                break
+        assert exists
```

Comment:
if that fails, one likely does not get much information about why it failed.

guess it would only say it is False instead of the expected True.

just a rough idea:
```
assert any(re.match(line, match_re) for line in lines)
```
check if that displays the match_re and also lines if it fails.

Reply:
Couldn't get it to spit out the re and the lines so I changed assert_line_exists to honor this approach.